machines/bleShare/request/requestMachine.ts

```diff
@@ -389,7 +389,7 @@ export const requestMachine =
               },
               on: {
                 DISMISS: {
-                  target: 'displayingIncomingVC',
+                  target: 'navigatingToHome',
```

Comment:
can we change it to navigationToHistory, as we are redirecting to history section.

Reply:
changed to navigationToHistory